pulsar-broker/src/main/java/com/yahoo/pulsar/broker/loadbalance/impl/ModularLoadManagerImpl.java

```diff
@@ -448,19 +445,34 @@ public void disableBroker() throws PulsarServerException {
      */
     @Override
     public synchronized void doLoadShedding() {
+        if (LoadManagerShared.isUnloadDisabledInLoadShedding(pulsar)) {
```

Comment:
is it possible to add unit-test case for `loadShedding` of `ModularLoadManagerImpl`?